src/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>To do list</title>
+		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+	</head>
+
+	<body>
+		<header></header>
+		<section class="to-do-section">
+			<div class="title padding-l-r">
+				<h4>
+					Today's To Do
+				</h4>
+				<button class="refresh-btn">
+					<i class="material-icons refresh-icon">refresh</i>
+				</button>
+			</div>
+			<hr>
+			<form action="#" id="form" class="padding-l-r">
+				<input type="text" placeholder="Add to your list ..." id="task" required class="task-input" />
+			</form>
+			<hr>
+			<div id="to-do-list">
+				
+			</div>
+
+			<button class="clear-btn">
+				Clear all completed
+			</button>
+		</section>
+	</body>
+
```

Comment:
- [OPTIONAL ] Follow our list of [best practices for HTML ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- Be careful with blank lines and indentation.
  - Do not add blank lines, spaces, or indentations without a reason.
  - For readability, add blank lines to separate large or logical code blocks.
  - For readability, add two spaces of indentation. Do not use the tab key